packages/bundle-size-checker/src/cli.js

```diff
@@ -159,6 +218,28 @@ yargs(process.argv.slice(2))
     },
     handler: run,
   })
+  // @ts-expect-error
```

Comment:
What's causing the error? In other scripts we usually are able to type yargs correctly.